proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -170,24 +170,30 @@ message Dispatch {
 }
 
 // Represents a dispatch, including its metadata
-message DispatchDetail {
-  // Unique identifier of the microgrid dispatch.
-  uint64 dispatch_id = 1;
+message Dispatch {
+  // Dispatch Metadata (id, create time, etc)
+  DispatchMetadata metadata = 1;
 
   // The dispatch data
-  Dispatch dispatch = 2;
+  DispatchData dispatch = 2;
```

Comment:
I would call this `data` now, to match `DispatchMetadata` -> `metadata`, `DispatchData` -> `data`, and so if we see a `dispatch` variable, we know is the composite message, so trying to name variables consistently `dispatch_metadata` and `dispatch_data` (unless it is already in a scope named *dispatch*, like in the composite `Dispatch` message, in that case it would be just redundant to add the `dispatch_` prefix).

```suggestion
  DispatchData data = 2;
```